.github/workflows/release-sdk.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       # All platforms: create wheels
       ##################################################
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.19.2
+        uses: pypa/cibuildwheel@v2.20.0
```

Comment:
please do a dry run with the release action as the last time we blindly updated the cbw version, something broke and we only learned that when doing the release.

Reply:
ok i won't merge it until the release. thanks for the heads up